homeassistant/components/google_assistant/smart_home.py

```diff
@@ -39,7 +39,7 @@
 # Mapping is [actions schema, primary trait, optional features]
 # optional is SUPPORT_* = (trait, command)
 MAPPING_COMPONENT = {
-    group.DOMAIN: [TYPE_SCENE, TRAIT_SCENE, None],
+    group.DOMAIN: [TYPE_LIGHT, TRAIT_ONOFF, None],
```

Comment:
Let's expose groups as switches for now: turn on and turn off. No reason to make them look like lights.

Reply:
Great. So now we will be able to turn on and off groups :)
Thanks